GA-worksheet.md

```diff
@@ -7,62 +7,62 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+The purpose of a header tag is for headers.
 2. What is a Navigation Bar?
+A list of links.
+3. What tag do we use to make a navigation bar? <ul  >
 
-3. What tag do we use to make a navigation bar? <      >
-
-4.  <ul> Stands for _____________________
-    <li> Stands for ______________________
+4.  <ul> Stands for _unordered lists____________________
+    <li> Stands for _list item_____________________
 
 5. What does the <a> tag do?
-
+links another page.
 6. What would the line of HTML below do?
 
 ``` html
 <a href="http://www.google.com">Google</a>
 ```
-
+it would open your browser.
```

Comment:
It would actually take you to http://www.google.com. It's a link:
`<a href="insert link here"> Name of the link </a>`
"href" is an attribute that tells the `<a>` or anchor tag where to go.